client/utils/gatsby/challenge-page-creator.js

```diff
@@ -120,7 +121,8 @@ function getProjectPreviewConfig(challenge, allChallengeEdges) {
   );
 
   return {
-    showProjectPreview: challengeOrder === 0 && usesMultifileEditor,
+    showProjectPreview:
+      challengeOrder === 0 && usesMultifileEditor && challengeType !== 14,
```

Comment:
Can we import and use `multiFileCertProject` from `challenge-types`?  It's a little clearer and DRYer

Reply:
added